safe-global/safe-dao-governance-app
# [EPIC] SEP #5 - Designs

## What it solves

Resolves SEP #5 design implementation

## How this PR fixes it

  - [x] [Intro page](https://www.figma.com/file/9nvicX7ng5528qAKXSf55J/Delegate-S-Tokens-App?type=design&node-id=2650-2290&mode=design&t=RpgG0U7UOCEwO40r-0)
  - [x] [Claim page](https://www.figma.com/file/igarLtnMhVtrYNOATmjJDc/Claim-S-Tokens-App?type=design&node-id=1503-8825&mode=design&t=gf59L0AavDWmeBi5-0)
  - [x] [Widget](https://www.figma.com/file/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?type=design&node-id=241-8837&mode=design&t=eAwhHaGwgFEYbWtB-0)

## How to test it

1. Open the app and observe the new design on the intro page.
2. Open the claim flow and observe the new design.
3. Open the `/widgets` route and observe the new widget design.

## Screenshots

![image](https://github.com/safe-global/safe-dao-governance-app/assets/20442784/e450d081-42dd-43fb-aa8f-f124fa4537d4)

![image](https://github.com/safe-global/safe-dao-governance-app/assets/20442784/34a46799-bee6-4ddc-b940-5dc26ff1c31c)

![image](https://github.com/safe-global/safe-dao-governance-app/assets/20442784/09328d55-7da1-40fd-9d8d-437864bdd99f)

## Checklist
